### PR TITLE
Add mention to dino.icu domains in scrapbook

### PR DESCRIPTION
**Pull request by javierzaleta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ Scrappy, our handy dandy [Slack bot](https://github.com/hackclub/scrappy), not o
 
 To put your profile on your own domain, run `/scrappy setdomain <domain>` in Slack, giving your website’s hostname (e.g. [`scrapbook.maggieliu.dev`](https://scrapbook.maggieliu.dev)). Then, add a `CNAME` record on your DNS provider, pointed to `cname.vercel-dns.com`. If you’re curious how this works, it’s [open source right here](http://github.com/hackclub/summer-domains).
 
+Hack Clubbers can also apply for a very cool `dino.icu` domain! You can use this domain for your scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above!
```
Comment (sampoder) [inline README.md:54]:
```suggestion
Hack Clubbers can also use a `dino.icu` subdomain! You can use this domain for your scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above.
```

File: README.md [hunk 2]
```diff
@@ -51,6 +51,8 @@ Scrappy, our handy dandy [Slack bot](https://github.com/hackclub/scrappy), not o
 
 To put your profile on your own domain, run `/scrappy setdomain <domain>` in Slack, giving your website’s hostname (e.g. [`scrapbook.maggieliu.dev`](https://scrapbook.maggieliu.dev)). Then, add a `CNAME` record on your DNS provider, pointed to `cname.vercel-dns.com`. If you’re curious how this works, it’s [open source right here](http://github.com/hackclub/summer-domains).
 
+Hack Clubbers can also use a `dino.icu` subdomain! You can use this domain for your scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above.
```
Comment (sampoder) [inline README.md:54]:
```suggestion
Hack Clubbers can also use a `dino.icu` subdomain! You can use this domain for your Scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above.
```

File: README.md [hunk 3]
```diff
@@ -51,6 +51,8 @@ Scrappy, our handy dandy [Slack bot](https://github.com/hackclub/scrappy), not o
 
 To put your profile on your own domain, run `/scrappy setdomain <domain>` in Slack, giving your website’s hostname (e.g. [`scrapbook.maggieliu.dev`](https://scrapbook.maggieliu.dev)). Then, add a `CNAME` record on your DNS provider, pointed to `cname.vercel-dns.com`. If you’re curious how this works, it’s [open source right here](http://github.com/hackclub/summer-domains).
 
+Hack Clubbers can also use a `dino.icu` subdomain! You can use this domain for your Scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above.
```
Comment (sampoder) [inline README.md:54]:
```suggestion
Hack Clubbers can also use a `dino.icu` subdomain by submitting a PR to our [DNS repository](https://github.com/hackclub/dns).
```

I'd like to shorten it a bit

Comment (sampoder) [reply README.md:54]: (ping me on slack after this)

File: pages/about.mdx [hunk 4]
```diff
@@ -59,6 +59,8 @@ Scrappy, our handy dandy [Slack bot](https://github.com/hackclub/scrappy), not o
 
 To put your profile on your own domain, run `/scrappy setdomain <domain>` in Slack, giving your website’s hostname (e.g. [`scrapbook.maggieliu.dev`](https://scrapbook.maggieliu.dev/)). Then, add a `CNAME` record on your DNS provider, pointed to `cname.vercel-dns.com`. If you’re curious how this works, it’s [open source right here](http://github.com/hackclub/summer-domains).
 
+Hack Clubbers can also use a `dino.icu` subdomain! You can use this domain for your Scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above.
```
Comment (sampoder) [inline pages/about.mdx:62]:
```suggestion
Hack Clubbers can also use a `dino.icu` subdomain by submitting a PR to our [DNS repository](https://github.com/hackclub/dns).
```

**Review by sampoder — APPROVED** [review]
Dino-tastic! Thanks as always!

**Comment by javierzaleta** [comment]
Sure Sam! As always I'm happy to help! Next thing is getting to work on V2! ⚡

**Pull request merged**